index.html

```diff
@@ -139,6 +139,23 @@ <h2>A project for first-time contributions</h2>
 
         </div>
         <!-- ________ Rakesh's card END ________  -->
+        
+        <!-- ________ Abhishek's card START ________  -->
+        <div class="card">
+
+            <p class="name">Abhishek Jadhav</p>
+            <p class="contact"><i class="fa fa-twitter"></i> <a href="https://twitter.com/Abhishek_7081" target="_blank">@Abhishek_7081</a></p>
+            <p class="about"> I do web development and machine learning. I am interestedin blockchain technology. </p>
+            <div class="resources">
+                <p>3 Useful Dev Resources</p>
+                <ul>
+                    <li><a href="https://www.youtube.com/watch?v=AAO4oq2M_48" target="_blank" title="This is a link to Quantum Machine Learning">Siraj does a good job always.</a></li>
+                    <li><a href="https://blockabitcoin.blogspot.com/" target="_blank" title="THis is about blockchain technlogy and bitcoin mining. </a></li>
```

Comment:
You have mixed the `title` and the `a` innertext. You will need to close the title with a `"` and the `a` tag with a `>` and then add the text to show on the card. The same as it is in the `li` above it.
```suggestion
                    <li><a href="https://blockabitcoin.blogspot.com/" target="_blank" title="Blockchain">Blockchain technlogy and bitcoin mining.</a></li>
```